test/aircraft/FlightManagementSystem/RouteModel.spec.js

```diff
@@ -965,6 +975,16 @@ ava('._createAmendedConvergentLeg() calls ._createLegsFromSidWaypointsAfterWaypo
     t.deepEqual(result, expectedResult);
 });
 
+ava('._createAmendedConvergentLeg() returns the leg unmodified when convergent waypoint is the STAR leg\'s first fix', (t) => {
+    const model = new RouteModel('DVC.GRNPA1.KLAS07R');
+    const waypointName = 'DVC';
+    const legIndex = 0;
+    const expectedResult = [model._legCollection[legIndex]];
```

Comment:
You should specify the expected result inline here instead of relying on the model to produce it for you.

Reply:
done